docker-compose.yml

```diff
@@ -1,5 +1,49 @@
 version: "3"
 services:
+  celestia-devnet:
+    container_name: celestia_devnet
+    image: ghcr.io/rollkit/local-celestia-devnet:v0.11.0-rc12
+    restart: on-failure
+    ports:
+      - "26657"
+      - "26658"
+      - "26659"
+      - "9090"
+
+  evm:
+    container_name: evm_base_shard
+    environment:
+      - DA_AUTH_TOKEN=${DA_AUTH_TOKEN:-}
+      - VALIDATOR_NAME=${VALIDATOR_NAME:-}
+      - CHAIN_ID=${CHAIN_ID-}
+      - KEY_NAME=${KEY_NAME:-}
+      - KEY_MNEMONIC=${KEY_MNEMONIC:-}
+      - KEY_BACKEND=${KEY_BACKEND:-}
+      - TOKEN_AMOUNT=${TOKEN_AMOUNT:-}
+      - STAKING_AMOUNT=${STAKING_AMOUNT:-}
+      - MIN_GAS_PRICE=${MIN_GAS_PRICE:-}
+      - TOKEN_DENOM=${TOKEN_DENOM:-}
+      - FAUCET_ADDR=${FAUCET_ADDR:-}
```

Comment:
I would prefix things that are only needed for dev/testing with `DEV_`. i think we should do this for both `FAUCET_ADDR` and `KEY_MNEMONIC/NAME/BACKEND` (in prod, we don't want any special account with a bunch of tokens loaded at genesis)

Reply:
is faucet_addr dev/testing only ? how do you imagine evm token distribution otherwise?

Reply:
fwiw, dev includes both devnet and testnet.

in mainnet, people would bridge eth into the chain and it would be minted then

Reply:
hmm, that seems like a pretty specific use case? i cant imagine every game project would want their chain to use bridged eth?

Reply:
also, these suggestions will need to happen upstream first. so if we want to add "dev" prefixes, lets make issue to do that in world engine